scripts/o3de/o3de/gem_properties.py

```diff
@@ -31,13 +33,13 @@ def update_values_in_key_list(existing_values: list, new_values: list or str, re
 
     returns updated existing value list
     """
-    if new_values:
+    if new_values is not None and new_values != []:
         new_values = new_values.split() if isinstance(new_values, str) else new_values
         existing_values.extend(new_values)
-    if remove_values:
+    if remove_values is not None and remove_values != []:
```

Comment:
This is overly verbose and not pythonic and doesn't work for empty string.

This should revert to the if new_values check

Reply:
Just want to double check that this only applies to `new_values` and `remove_values` correct? `replace_values` needs the more verbose check in the event the user wishes to clear a list by supplying an empty list `[]`.

Reply:
Yes, this only applies to `new_values` and `remove_values` variables